bin/bootstrax

```diff
@@ -200,12 +200,18 @@ dead_state = 'dead_bootstrax'
 
 # The maximum time difference (s) allowed between the timestamps in the data and the
 # duration of the run (from the runs metadeta). Fail if the difference is larger than:
-max_timetamp_diff = 2
+max_timetamp_diff = 5
 
 # The maximum number of retries for processing a run. After this many times of retrying
 # to process a run, the DAQ-group has to either manually fix this run or manually fail it.
 max_n_retry = 20
 
+# Bootstrax retries runs multiple times. If there have been this many fails we could lower
+# the resources to be somewhat more lenient on the CPU and RAM of the eventbuilders. Use
+# this option with care as we might be running in a sub-optimal mode that may not be
+# noticed and eventbuilders may be spending more time on trying to reprocess failed runs.
+lower_resources_after_n_failures = 10
```

Comment:
A nice-to-have feature would be something that can extract some useful information from the error message and decide if reducing resources would actually solve the issue. Very far outside the scope of this PR, but maybe something for a rainy week.

Reply:
Thanks for the idea! Of course this is somewhat hard to know a-priori as CPU and RAM issues have been throwing some quite funny messages because of multiprocessing. Might be nice to indeed keep in mind though.